AEPServices/Sources/utility/unzip/URL+Validator.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+
+extension URL {
+    ///
+    /// Validates a URL against a Zip Slip attack. Simply checks if there is any sort of traversal attempted in the url
+    /// - Returns: True if valid, false if invalid
+    func isValidUrl() -> Bool {
```

Comment:
change the method name to be more specific

Reply:
lmk if this name works.